core/modules/system/system.theme.inc

```diff
@@ -406,12 +406,18 @@ function theme_admin_page($variables) {
 
   $output = '<div class="admin clearfix">';
```

Comment:
Looks like this line should be removed as it's repeated below. Otherwise we'll have an unclosed div.

Reply:
Indeed. Fixed.